build/webpack-base-config.js

```diff
@@ -45,7 +45,7 @@ const webpackBaseConfig = {
 				loader: 'babel-loader',
 			},
 			{
-				test: /\.css$/,
+				test: /\.(post)?css$/,
```

Comment:
I appreciate the IDE compatibility improvement but would prefer to move this out and address it holistically in a separate Issue/PR for further discussion.

----

Based on https://github.com/johnsoncodehk/volar/issues/765, it seems the recommendation is to use https://marketplace.visualstudio.com/items?itemName=cpylua.language-postcss with the following configuration to make it work with `.css` extensions:
```
  "files.associations": {
    "*.css": "postcss"
  }
```